cli/internal/terraform/terraform.go

```diff
@@ -170,6 +171,44 @@ func (c *Client) ShowIAM(ctx context.Context, provider cloudprovider.Provider) (
 	}
 }
 
+// ShowInfrastructure reads the state of Constellation cluster resources from Terraform.
+func (c *Client) ShowInfrastructure(ctx context.Context, provider cloudprovider.Provider) (state.Infrastructure, error) {
```

Comment:
Is there still a reason to keep the old `ShowCluster` (and also the `CreateCluster`) function which returns `ApplyOutput`, instead of letting the terraform package just return `state.Infrastructure`?

Reply:
Nvm, there is already a PR for that